.github/workflows/node.js.yml

```diff
@@ -0,0 +1,21 @@
+name: Node.js CI
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js 16.x
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16.x
+        cache: 'npm'
+    - run: npm ci
+    - run: npm run build --if-present
+    - name: Run tests
+      run: npm test
+      
```

Comment:
Last line should not have any spaces, or any character. It should be just an empty line generated by the line break, representing the EOF

Reply:
So, what should I do? 
Delete the file and start all over again? or Can I edit this file?

Reply:
I recommend you to pull this branch locally, continue working there and edit all the files locally. Then commit and push to this branch